finality-aleph/src/lib.rs

```diff
@@ -135,6 +137,23 @@ where
 {
 }
 
+pub trait NumberOps:
+    std::ops::Add<Output = Self>
+    + std::ops::Sub<Output = Self>
+    + std::ops::Mul<Output = Self>
+    + From<u32>
+{
+}
+
+impl<T> NumberOps for T
+where
+    T: std::ops::Add<Output = Self>,
+    T: std::ops::Sub<Output = Self>,
+    T: std::ops::Mul<Output = Self>,
+    T: From<u32>,
+{
+}
```

Comment:
Where do we actually use math operations on block numbers?

Reply:
Nowhere, this was leftovers from when I needed it.